recipe/build.sh

```diff
@@ -19,5 +21,20 @@ export INSTALL_INCLUDE="${PREFIX}/include"
 export BLAS="-lopenblas"
 export LAPACK="-lopenblas"
 
+# (optional) write out various make variables for easier build debugging
+eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make config 2>&1 | tee make_config.txt
+
 eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make -j1
+
+# make install below fails to link libmetis unless it is copied to $PREFIX/lib
+cp $SRC_DIR/lib/libmetis$DYNAMIC_EXT $PREFIX/lib
```

Comment:
Maybe we could just build METIS first and install it. Then build everything else and install them. It might save us a few ugly hacks that cause us problems later.